### ISSUE TITLE
Regarding the issue of self._normalize_input

### ISSUE THREAD

**Issue by MM666-code** [CityMind-Lab/ICML25-TimeVLM#4]
Thank you very much for providing the source code of the paper. I am very interested in this open-source code, but I have a question to ask you. Since the data has already been standardized before entering the model training, why do we need to go through the following operations before converting the data into images：

    def _normalize_input(self, x):
        means = x.mean(1, keepdim=True).detach()
        x = x - means
        stdev = torch.sqrt(torch.var(x, dim=1, keepdim=True, unbiased=False) + 1e-5)
        stdev /= self.config.norm_const
        x = x / stdev
        return x, means, stdev

I hope to receive your reply. Wishing the author all the best.
